newsletter-february-2023.rst

```diff
@@ -80,34 +71,43 @@ Upcoming features
 
 Interested in all the details? `View our full Roadmap 📍️`_
 
-.. _View our full Roadmap 📍️: https://github.com/orgs/readthedocs/projects/156/views/4
+.. _View our full Roadmap 📍️: https://github.com/orgs/readthedocs/projects/156/views/1
 
 Possible issues
 ---------------
 
-* The change of the standard build output directory will cause issues for anyone that has custom builds and have guessed the old unofficial output directory.
+- 🚦️ There is a schedule maintenance window on Cloudflare that provides our CDN and SSL provisioning. 
+  We are working to ensure that nothing is offline other than creation of new domains:
+
+    On February 14th, 2023,
+    Cloudflare will be doing database maintenance that will impact SSL API availability and may result in certificate issuance delays.
+    The scheduled maintenance will be on February 14, 2023, 14:00 - 16:00 UTC.
+
+- The change of the standard build output directory will cause issues for anyone that has custom builds and have guessed the old unofficial output directory.
   See technical details in `readthedocs.org#9888`_
-* If you want to use Sphinx 6 wth sphinx-rtd-theme,
-  we will ship an update next week of both the new theme and an update on our platform dealing with the removal of jQuery from Sphinx.
+- If you want to use Sphinx 6 with sphinx-rtd-theme,
+  we have shipped an update of both the new theme and an update on our platform dealing with the removal of jQuery from Sphinx.
   See also :doc:`/sphinx6-upgrade`.
```

Comment:
Should we update this post with this information as well? A lot of users looking to upgrade will land there. 

Not a blocker for this PR, but another PR instead.

Reply:
That's a great idea!